examples/with-strict-csp/middleware.js

```diff
@@ -0,0 +1,34 @@
+import { NextResponse } from 'next/server'
+
+export function middleware(request) {
+  const nonce = crypto.randomUUID()
+  const cspHeader = `
+    default-src 'self';
+    script-src 'self' 'nonce-${nonce}' 'strict-dynamic';
+    style-src 'self' 'nonce-${nonce}';
+    img-src 'self' blob: data:;
+    font-src 'self';
+    object-src 'none';
+    base-uri 'self';
+    form-action 'self';
+    frame-ancestors 'none';
+    block-all-mixed-content;
+    upgrade-insecure-requests;
+`
+
+  const requestHeaders = new Headers(request.headers)
```

Comment:
```suggestion
  const requestHeaders = new Headers()
```
Same as https://github.com/vercel/next.js/pull/54601/files#r1306561406